packages/jest-environment/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jest-environment-hops",
+  "version": "14.0.0-nightly.4",
+  "description": "Opinionated Jest environment to run Puppeteer-powered tests against Hops apps",
+  "keywords": [
+    "hops",
+    "jest"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": "12 || 14 || 15"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/xing/hops.git",
+    "directory": "packages/jest-environment"
+  },
+  "main": "index.js",
+  "dependencies": {
+    "debug": "^4.1.1",
+    "eprom": "^1.0.0",
+    "fs-extra": "^9.0.1",
+    "jest-environment-node": "^26.6.2",
```

Comment:
Having this as a direct dependency might bring us trouble with updated Jest major versions again.
We should probably implement it the same way as we did here: https://github.com/xing/hops/blob/master/packages/jest-preset/jest-preset.js#L2-L18 or what do you think?

Reply:
The dependency to `jest-environment-node` should be removed here.